HoneyBadgers.WebApp/Controllers/ReportController.cs

```diff
@@ -1,43 +1,61 @@
 ﻿using Microsoft.AspNetCore.Mvc;
-using System;
-using System.Collections.Generic;
-using System.Composition;
-using System.Diagnostics;
-using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
 using HoneyBadgers.Entity.Models;
+using HoneyBadgers.Logic.Services.Interfaces;
 using HoneyBadgers.WebApp.Models;
 using Newtonsoft.Json;
 
 namespace HoneyBadgers.WebApp.Controllers
 {
     public class ReportController : Controller
     {
-        private readonly IHttpClientFactory _iHttpClientFactory;
-        public ReportController(IHttpClientFactory iHttpClientFactory)
+        private readonly IHttpClientFactory _httpClientFactory;
+        private readonly IReportService _reportService;
+        public ReportController(IHttpClientFactory httpClientFactory, IReportService reportService)
         {
-            _iHttpClientFactory = iHttpClientFactory;
+            _httpClientFactory = httpClientFactory;
+            _reportService = reportService;
         }
         [HttpGet]
         public async Task<ActionResult<Report>> Index()
         {
-            var client = _iHttpClientFactory.CreateClient();
-            var request = new HttpRequestMessage(HttpMethod.Get, "https://localhost:5001/api/reports");
+            // var client = _iHttpClientFactory.CreateClient();
+            // var request = new HttpRequestMessage(HttpMethod.Get, "https://localhost:5001/api/reports");
+            //
+            // HttpResponseMessage result;
+            // try
+            // {
+            //     result = await client.SendAsync(request);
+            // }
+            // catch(Exception ex)
+            // {
+            //     return View(new Report[]{ new Report(ex.Message) });
+            // }
+            // var content = await result.Content.ReadAsStringAsync();
+            // var json = JsonConvert.DeserializeObject<Report[]>(content); 
+            return View();
```

Comment:
sporo zakomentowanego kodu :O?

Reply:
pisałam na slacku, że zostawiam bo jest tam obsługa błędu kiedy rest api nie odpowiada tylko teraz to nie zadziała i będę to przerabiać kiedy będę dostawać autentyczne dane z rest api